Gemfile

```diff
@@ -14,11 +14,7 @@ gem 'gds-sso', '~> 13.5.0'
 gem 'govuk_admin_template'
 gem "govuk_app_config", "~> 0.2.0"
 
-if ENV["API_DEV"]
```

Comment:
Why no more dev option?

Reply:
It is now considered bad practise and has been removed from most of the other app (as far as I am aware)